From eb8ef0c1ee53d2f2eb16edd11e94745b570b80b9 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Wed, 27 Mar 2024 07:56:18 -0300
Subject: [PATCH] Build make changos

Don't handle apns-conf
Replace Camera2 with Aperture
Fix build name scheming

Change-Id: I0cf3304016479585005d437e322c992f4b8a2d77
---
 target/product/full_base_telephony.mk | 5 -----
 target/product/handheld_product.mk    | 2 +-
 tools/buildinfo.sh                    | 2 +-
 3 files changed, 2 insertions(+), 7 deletions(-)

diff --git a/target/product/full_base_telephony.mk b/target/product/full_base_telephony.mk
index a37fd48af..5470ad2be 100644
--- a/target/product/full_base_telephony.mk
+++ b/target/product/full_base_telephony.mk
@@ -26,10 +26,5 @@ PRODUCT_VENDOR_PROPERTIES := \
 PRODUCT_COPY_FILES := \
     frameworks/native/data/etc/handheld_core_hardware.xml:$(TARGET_COPY_OUT_VENDOR)/etc/permissions/handheld_core_hardware.xml
 
-ifeq ($(CUSTOM_BUILD),)
-PRODUCT_COPY_FILES += \
-    device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
-endif
-
 $(call inherit-product, $(SRC_TARGET_DIR)/product/aosp_base.mk)
 $(call inherit-product, $(SRC_TARGET_DIR)/product/telephony.mk)
diff --git a/target/product/handheld_product.mk b/target/product/handheld_product.mk
index c3cd30e94..fee227955 100644
--- a/target/product/handheld_product.mk
+++ b/target/product/handheld_product.mk
@@ -36,7 +36,7 @@ PRODUCT_PACKAGES += \
 
 ifeq ($(CUSTOM_BUILD),)
 PRODUCT_PACKAGES += \
-    Camera2 \
+    Aperture \
     LatinIME
 endif
 
diff --git a/tools/buildinfo.sh b/tools/buildinfo.sh
index 31639c03c..8dfca927d 100755
--- a/tools/buildinfo.sh
+++ b/tools/buildinfo.sh
@@ -57,6 +57,6 @@ if [ -n "$BUILD_THUMBPRINT" ] ; then
   echo "ro.build.thumbprint=$BUILD_THUMBPRINT"
 fi
 
-echo "org.pixelexperience.device=$CUSTOM_DEVICE"
+echo "org.pixelexperience.device=$TARGET_PRODUCT"
 
 echo "# end build properties"
-- 
2.34.1

